### PR TITLE
css-two-column-layout exercise - Heerak Moon

### PR DESCRIPTION
**Pull request by hmoon2**
css-two-column-layout

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      <h1>SCALE, SPEED, AGILITY</h1>
+      <p>We've dedicated thousands of man hours to making healthcare
+        institutions in the Helium network run 10x more efficiently
+        and deliver an unparalleled quality of care.
+      </p>
+    </div>
+    <div class="row">
+      <div>
+        <img src="images/software.svg" alt="Africa">
+      </div>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:24]: No column class? Why?

File: css-two-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      <div class="col-two-thirds">
+        <h3>INTUITIVE, SMART SOFTWARE</h3>
+        <p>We build products that transform how healthcare facilities
+          run across Africa &mdash; making them radically more efficient operationally
+          and financially, a complete turnaround.
+        </p>
+      </div>
+      <div>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:32]: No column class? Why?

File: css-two-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      <div class="col-two-thirds">
+        <h3>PRACTICE MEDICINE, NOT ADMINISTRATION</h3>
+        <p>Our all-in-one electronic health records (EHR), hospital management
+          (HMIS) &amp; billing system helps streamline your facility, reduce waste
+          and improve quality of care.
+        </p>
+      </div>
+    </div>
+    <div class="row">
+      <div>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:44]: No column class? Why?

File: css-two-column-layout/index.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        <img src="images/circumstance.svg" alt="electron rings">
+      </div>
+      <div class="col-two-thirds">
+        <h3>FOR ANY CIRCUMSTANCE</h3>
+        <p>Our intuitive system is built around the peculiarities of your
+          facility; we support offline deployments, sync to the cloud and run
+          on any device 24/7.
+        </p>
+      </div>
+      <div>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:54]: No column class? Why?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by hmoon2** [comment]
css-two-column-layout-changes

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by hmoon2** [comment]
css-two-column-layout-resubmit

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**